component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -1525,6 +1535,42 @@ public Response importUsers(@Context HttpServletRequest request,
     return errorResponse == null ? Response.noContent().build() : errorResponse;
   }
 
+  @GET
+  @Path("userCardSettings")
+  @Produces(MediaType.APPLICATION_JSON)
+  @RolesAllowed("users")
+  @Operation(
+          summary = "Gets the filed settings of a user card",
+          description = "Gets the filed settings of a user card",
```

Comment:
field

Reply:
Okay so this code are not really clear to understand 

Please specify what we are to do here